contracts/LockedNORI.sol

```diff
@@ -734,4 +744,30 @@ contract LockedNORI is
     }
     _burn(account, amount);
   }
+
+  /**
+   * @dev See {IERC777-operatorBurn}. Reverts, unsupported.
+   *
+   * Emits {Burned} and {IERC20-Transfer} events.
+   */
+  function operatorBurn(
+    address,
+    uint256,
+    bytes memory,
+    bytes memory
+  ) public virtual override {
+    revert();
```

Comment:
I think the base branch has a better variant of this override with a revert message